src/components/Viewport.ts

```diff
@@ -1849,4 +1923,39 @@ export default class Viewport {
 
     return { removedPanels, addedPanels };
   }
+
+  private resizePanels(panels: Panel[]): void {
+    const options = this.options;
+    const panelBboxes = this.panelBboxes;
+
+    if (options.isEqualSize === true) {
```

Comment:
isEqualSize is string[] type automatically.

```ts
const isEqualSize = this.options.isEqualSize;

if (isEqualSize) {
    if (isEqualSize === true) {
    } else {
      // isEqualSize is string[] automatically
    }
}

Reply:
Looks like I just copy-pasted that, just deleting `as` worked for me.
```ts
    if (options.isEqualSize === true) {
      //
    } else if (options.isEqualSize) {
      // isEqualSize is string[]
    }
```